service/src/main/java/com/bankbazaar/kafka/service/consumer/KafkaConsumer.java

```diff
@@ -1,32 +1,72 @@
 package com.bankbazaar.kafka.service.consumer;
 
 import com.bankbazaar.kafka.core.model.Data;
+import com.bankbazaar.kafka.dto.model.FileStatusDto;
+import com.bankbazaar.kafka.service.service.FileStatusService;
 import com.opencsv.CSVWriter;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.kafka.annotation.KafkaListener;
+import org.apache.kafka.streams.kstream.KStream;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.retry.support.RetryTemplate;
 import org.springframework.stereotype.Service;
 
+import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.io.Writer;
+import java.util.function.Consumer;
 
 @Slf4j
 @Service
 public class KafkaConsumer {
```

Comment:
Rename to KafkaStreamsConfig. This class no longer is a single kafka consumer

Reply:
all KafkaStream bean moved to KafkaStreamsConfig